docs/configuration/README.md

```diff
@@ -1,7 +1,27 @@
 # Configuration
 
-`elife-xpub` can be configured using multiple different methods. The configuration methods, and their precedence, are as follows:
+`elife-xpub` can be configured using multiple different methods. The main method of configuration uses [node-config](https://github.com/lorenwest/node-config). 
+
+
+The configuration methods for `node-config`, and their precedence, are as follows:
 
 - `config/default.js` is used as the base configuration
 - `config/${environment}.js` is used for environment specific configuration, which can be stored publically. The environment is specified by setting `NODE_CONFIG_ENV` to the appropriate environment.
 - `config/custom-environment-variables.js` is used for environment specific secrets. This gets populated using configuration within [builder-configuration](https://github.com/elifesciences/builder-configuration), and injected into the app using [elife-xpub-formula](https://github.com/elifesciences/elife-xpub-formula) and [elife-xpub-deployment](https://github.com/elifesciences/elife-xpub-deployment).
+
+
+### Other methods
+
+Other config methods are used for specific functionality, including:
+
+- [pubsweet-components](https://gitlab.coko.foundation/pubsweet/pubsweet/tree/master/packages/components) - used for loading server-side components

```

Comment:
Maybe write the filename where you specify the config for these (e.g. `config/components.json`) and explain how to change this file? The link you provided doesn't show me how to change it. Also, wasn't this for all type of components both frontend and backend?

Reply:
The only example on `components.json` I found is in the book here (page 82)
https://coko.foundation/wp-content/uploads/2018/07/Coko_Pubsweet-screen.pdf

Reply:
Thanks, I couldn't find the book myself.

Reply:
I'm hesitant to put more code references that what's already there, since it's stuff that would have to be updated/maintained if we were to change anything related to pubsweet configuration. I'm happy to state that all config files live inside `config/`, and linking to the official docs should be enough for people to find the relevant files.